### PR TITLE
Delete old instructions from package upload docs

### PR DESCRIPTION
**Pull request by ktully**
Remove documentation references to manually uploading package to website, as this functionality has been disabled.

### CODE REVIEW

File: chocolatey/Website/Views/Courses/CreatingChocolateyPackages/BuildingTestingAndPushing.cshtml [hunk 1]
```diff
@@ -97,4 +96,4 @@
     }
 </div>
 
-@Html.Partial("~/Views/Courses/_Popup.cshtml", Model)
\ No newline at end of file
+@Html.Partial("~/Views/Courses/_Popup.cshtml", Model)
```
Comment (ktully) [inline chocolatey/Website/Views/Courses/CreatingChocolateyPackages/BuildingTestingAndPushing.cshtml:99]: This wasn't me; I was lazy and edited directly in Github and it looks like the Github browser editor added a terminal line ending.

**Comment by gep13** [comment]
@ktully thank you for taking the time to submit this PR.

There is a file in the root of the Views/Documentation folder (https://github.com/chocolatey/chocolatey.org/blob/master/chocolatey/Website/Views/Documentation/_README.md) to explain that the files are not maintained in this repository, but rather in the choco-wiki repository here:

https://github.com/chocolatey/choco-wiki/blob/master/CreatePackages.md

Can you please update this PR to only include the modification to the cshtml file, leaving the .md file unchanged, and then we can update that file in the right place.

Thanks

**Comment by ktully** [comment]
@gep13 done, thanks for the feedback. 

Sorry, I missed the README stating these originated elsewhere.

**Comment by gep13** [comment]
@ktully no need to apologise.  Thanks again for catching this, and taking the time to update our docs!

**Pull request merged**